### PR TITLE
man Dockerfile ADD/COPY/FROM clarify

### PR DESCRIPTION
**Pull request by sallyom**
Signed-off-by: Sally O'Malley somalley@redhat.com

### CODE REVIEW

File: man/Dockerfile.5.md [hunk 1]
```diff
@@ -150,13 +156,20 @@ A Dockerfile is similar to a Makefile.
   the image.
 
 **LABEL**
-  -- `LABEL <key>[=<value>] [<key>[=<value>] ...]`
+  -- `LABEL <key>[=<value>] [<key>[=<value>] ...]`or 
+  ```
+  LABEL <key>[ <value>]
+  LABEL [<key>[ <value>]
```
Comment (phemmer) [inline man/Dockerfile.5.md]: looks like an errant `[` before `<key>`

Comment (sallyom) [inline man/Dockerfile.5.md]: fixed, thanks

----- Original Message -----

**Comment by SvenDowideit** [comment]
LGTM - @thaJeztah @moxiegirl

**Comment by thaJeztah** [comment]
Great! Thanks again @sallyom, LGTM

**Pull request merged**